config/initializers/assets.rb

```diff
@@ -9,3 +9,13 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 # Rails.application.config.assets.precompile += %w( search.js )
+
+Rails.application.config.assets.precompile += %w( animate.min.css )
+Rails.application.config.assets.precompile += %w( creative.css )
+
+
+Rails.application.config.assets.precompile += %w( jquery.js )
+Rails.application.config.assets.precompile += %w( jqueryeasing.min.js )
+Rails.application.config.assets.precompile += %w( jquery.fittext.js )
+Rails.application.config.assets.precompile += %w( wow.min.js )
+Rails.application.config.assets.precompile += %w( creative.js )
```

Comment:
I think no. I do not remember having done it. It should have been at the beginning.